cranelift-codegen/src/verifier/mod.rs

```diff
@@ -468,6 +468,41 @@ impl<'a> Verifier<'a> {
         Ok(())
     }
 
+    /// Check that the given EBB can be encoded as a BB, by checking that only
+    /// branching instructions are ending the EBB.
+    fn encodable_as_bb(
+        &self,
+        ebb: Ebb,
+        errors: &mut VerifierErrors,
+    ) -> VerifierStepResult<()> {
+        // Skip this verification if the environment variable is not set.
+        if let Err(_) = std::env::var("CRANELIFT_BB") {
```

Comment:
Maybe make this a property of the Verifier itself, so it's not doing a syscall per ebb?

Reply:
Reading env vars doesn't involve a trip into the kernel, at least on Linux .. pointers to them are pushed on the stack before main() is entered.